util-tunable/src/main/scala/com/twitter/util/tunable/TunableMap.scala

```diff
@@ -133,7 +134,7 @@ object TunableMap {
   }
 
   object Key {
-    def apply[T](id: String)(implicit m: Manifest[T]): Key[T] =
+    def apply[T](id: String)(implicit m: ClassTag[T]): Key[T] =
```

Comment:
`Manifest` no longer exists in Scala 3 and there is no equivalent replacement.

Since it is only used to retrieve the runtime class (which is erased anyway), `ClassTag` should work the same way here, as far as I understand it.